src/Facing.php

```diff
@@ -159,17 +149,17 @@ public static function validate(int $facing) : void{
 	}
```

Comment:
There's no point deprecating this. We're making sweeping BC breaks here anyway.

Reply:
So i remove this function ?